FinniversKit/Sources/Recycling/GridViews/Markets/GridView/Cell/MarketsGridViewCell.swift

```diff
@@ -8,6 +8,8 @@ class MarketsGridViewCell: UICollectionViewCell {
     // MARK: - Internal properties
 
     private let cornerRadius: CGFloat = 16
+    
+    var isFinn: Bool = true
```

Comment:
i'm not 100% sure you need this
if you can init somehow the cell with `numberOfLines = 2`, you could perhaps check that to figure out the sizing